server/src/main/kotlin/com/hockey/elo/elotracker/match/service/MatchService.kt

```diff
@@ -25,19 +25,25 @@ class MatchService(private val matchRepository: MatchRepository) {
       match.playerTwoScore = matchUpdateRequest.playerTwoScore
       matchRepository.save(match)
     } else {
-      throw MatchNotFoundException("MatchRecord Id: $id Not Found")
+      throw MatchNotFound("MatchRecord Id: $id Not Found")
     }
-
   }
 
-  fun completeMatch(id: Long, playerOneWin: Boolean) {
+  fun completeMatch(id: Long, playerOneDidWin: Boolean) {
     if (matchRepository.findById(id).isPresent) {
       val match = matchRepository.findById(id).get()
-      if (playerOneWin) match.winner = match.playerOneId else match.winner = match.playerTwoId
+      if (playerOneDidWin) {
+        match.winner = match.playerOneId
```

Comment:
The `score` attribute of the match resource was meant to hold the score of the match for match history viewing.z

My earlier comment meant that we should be updating the elo of the actual user resource.

Reply:
ok gotcha. That makes sense.